src/zenml/zen_stores/base_zen_store.py

```diff
@@ -826,15 +702,13 @@ def _default_workspace(self) -> WorkspaceResponse:
                 f"The default workspace '{workspace_name}' is not configured"
             )
 
+    @abstractmethod
     def _create_default_workspace(self) -> WorkspaceResponse:
         """Creates a default workspace.
 
         Returns:
             The default workspace.
         """
-        workspace_name = self._default_workspace_name
-        logger.info(f"Creating default workspace '{workspace_name}' ...")
-        return self.create_workspace(WorkspaceRequest(name=workspace_name))
```

Comment:
This is what I meant earlier: it now also makes sense that the default stack (and maybe even the default user) are also only created in the SQL zen store.

Reply:
I've moved all the database initialization into the SQLZenStore as you suggested, much better :)